lib/beaker-answers/versions/version20172.rb

```diff
@@ -19,14 +19,13 @@ def generate_hiera_config
         # We're removing the flattened keys added in the Version20162 and
         # reading them here rather than breaking compatibilty with existing
         # Version20162 behavior. We're sorry.
-        hiera_hash.reject! do |k,v|
+        hiera_hash.reject! do |k, _v|
           flatten_keys_to_joined_string(@options[:answers]).include?(k)
         end
-        stringified_answers = @options[:answers].inject({}) do |hash,entry|
+        stringified_answers = @options[:answers].each_with_object({}) do |entry, hash|
```

Comment:
Let's drop this one; you can use each_with_object() if you want, but let's not force it where inject, reduce, etc. have meaning outside of ruby.

Reply:
https://rubocop.readthedocs.io/en/latest/cops_style/#styleeachwithobject disabled